bin/install

```diff
@@ -18,23 +19,109 @@ error_exit() {
   exit "${2:-1}"
 }
 
-platform() {
+get_platform() {
```

Comment:
Renamed the `platform()` function to `get_platform()`, in line with other functions added in the PR